src/main/resources/application.conf

```diff
@@ -105,5 +105,6 @@ userprofile {
 
 nih {
   whitelistFile = "users.txt"
-  rawlsGroupName = "dbGapAuthorizedUsers"
+  rawlsGroupName = "testRealm"
+//  rawlsGroupName = "dbGapAuthorizedUsers"
```

Comment:
Rawls is still giving me a 500 with "Cannot find Group corresponding to RawlsGroupName(dbGapAuthorizedUsers)"